src/Symfony/Component/AssetMapper/AssetMapperRepository.php

```diff
@@ -55,7 +55,7 @@ public function find(string $logicalPath): ?string
 
             $file = rtrim($path, '/').'/'.$localLogicalPath;
             if (file_exists($file)) {
-                return $file;
+                return realpath($file);
```

Comment:
Using `realpath` everywhere now in this file so that all paths use the "real" path. That's just "nice" because you'll get back full paths - instead of potentially getting back file paths like `/var/something/else/../other-path/app.js` - and is important especially for `findLogicalPath()` where we pass in a `$filesystemPath` and then do some matching by the filesystem path.